screens/DetailedMuseumScreen.tsx

```diff
@@ -4,45 +4,116 @@ import {
   StyleSheet,
   Dimensions,
   Image,
+  Text,
+  Animated,
 } from 'react-native';
 import type {NativeStackScreenProps} from '@react-navigation/native-stack';
+import {Chip} from '@rneui/themed';
+import {useState} from 'react';
+import {startCase} from 'lodash';
 
 import {MOCK_IMAGE} from '../utils/constans';
 import type {MainRootStackParamList} from '../App';
 import {getAppStyles} from '../utils/styles';
 
+import ScrollView = Animated.ScrollView;
+
+import {getAppColors} from '../utils/colors';
+
 type Props = NativeStackScreenProps<MainRootStackParamList, 'DetailedMuseum'>;
 
 export default function DetailedMuseumScreen({route}: Props) {
   const colorScheme = useColorScheme();
+  const [selectedField, setSelectedField] = useState<string>(
```

Comment:
The typing could be better here if we know the field names beforehand - it could be attached to `fieldsToDisplay` somehow, the type could look like:

`type DisplayFields = "country" | "city"...`

and then use it to type the rest